Dockerfile.hack

```diff
@@ -0,0 +1,7 @@
+FROM registry.replicated.com/library/statsd-graphite:1.1.8-20211119
```

Comment:
so in other words we're taking the existing image and just updating deps instead of changing the binary?

This is for-sure going to bite us in the ass at some point

Reply:
yes....

Reply:
also, there isn't _the_ binary...  this image runs supervisor with 3 different services

Reply:
fair enough 😆